internal/git/releases.go

```diff
@@ -106,8 +108,21 @@ func (g *Git) CreateRelease(releaseInfo releases.ReleasesInfo, outputs map[strin
 					// TODO: Consider deleting and recreating the release if we are moving forward with publishing
 					return nil
 				}
+				// Go has no publishing job, so we publish a CLI event on github release here
+				if lang == "go" {
+					if publishEventErr := actions.TriggerPublishingEvent(info.Path, "failed", "go"); publishEventErr != nil {
+						fmt.Printf("failed to write publishing event: %v\n", publishEventErr)
+					}
+				}
 
 				return fmt.Errorf("failed to create release for tag %s: %w", *tagName, err)
+			} else {
+				// Go has no publishing job, so we publish a CLI event on github release here
+				if lang == "go" {
```

Comment:
Seems odd that this is duplicated

Reply:
Ya wasn't a super clean way if not duplicating since we return. There's some thing I could do to abstract out but it felt weirded